### DISCUSSION TITLE
如何与wiki对接oauth2呢

### DISCUSSION THREAD

**Discussion by vlssu** [bs-community/blessing-skin-server#339] [Q&A]
我这么设置对吗，我实际操作下来发现我失败了，是我设置有什么疏忽错误的地方吗
我是想让wiki使用皮肤站登录
https://js.wiki/
![image](https://user-images.githubusercontent.com/43847794/131427899-9f56e6dd-da6f-4e88-b948-8ec0024c4df5.png)
![image](https://user-images.githubusercontent.com/43847794/131427943-965fcf7a-c470-4aa6-9b9d-2681e948475f.png)

**Comment by ghost** [comment]
你试试用Generic OAuth2 (Generic OAuth2)

**Comment by ghost** [comment]
我使用Generic OAuth2 (Generic OAuth2)可以得到请求
![A](https://user-images.githubusercontent.com/82318992/151365743-ec87e261-1de2-459c-8de0-352ebf122624.png)

**Reply by vlssu** [reply]
不，请求是可以收到，但返回值是缺少的，wiki那边收到的信息不全，是无法在Wiki上创建一个新用户
我也写了新的，那边是新版的可能更直观？https://github.com/bs-community/blessing-skin-server/discussions/357

**Reply by ghost** [reply]
获取用户信息的端点是api/user,但是它要包含token才可，不然获取不到用户信息

**Reply by ghost** [reply]
我尝试模拟，会json解析错误

**Reply by vlssu** [reply]
> 我尝试模拟，会json解析错误

这样只能等待作者回复了

**Reply by ghost** [reply]
我这也是信息不全
